public/script.js

```diff
@@ -40,28 +40,48 @@ function getSnippet() {
             } else if (data === "") {
                 alert('Unknown  error has occurred.')
             } else {
-                // sending data and speaking .
+                paragraph.innerHTML = data;
                 speech.text = data;
-                window.speechSynthesis.speak(speech)
-                paragraph.innerHTML = data
-                setSucces(inputs)
+                
+            if (speechMuted) {
```

Comment:
The name doesn't make sense "If speech is muted then read the speech"? Fix the logic